src/daemon/utils_cache.h

```diff
@@ -114,6 +114,8 @@ int uc_iterator_get_meta(uc_iter_t *iter, meta_data_t **ret_meta);
  */
 int uc_meta_data_exists(const value_list_t *vl, const char *key);
 int uc_meta_data_delete(const value_list_t *vl, const char *key);
+/* Same API as meta_data_toc. */
+int uc_meta_data_toc(const value_list_t *vl, char ***toc);
```

Comment:
What does `toc` mean here? table of content? I think the readers won't have enough context to know what it means.

Reply:
That's correct. I'd like to keep it; it's a name used in collectd together with metadata already, see `meta_data_toc` in meta_data.h in this directory. Note that I sent the same change upstream.